src/components/Filter/Filter.tsx

```diff
@@ -80,27 +128,23 @@ const Filter: React.FC<Props & TestProps> = props => {
           </span>
         </button>
         {open && (
-          <div css={optionsStyle({})(theme)} data-testid="filter-menu">
-            <button
-              type="button"
-              css={defaultOptionStyle(defaultValue, selectedItem)(theme)}
-              onClick={() => {handleSelect(defaultValue);}}
-            >
-              {defaultValue.label}
-            </button>
-            {items &&
-              items
-                .filter(option => option.value !== defaultValue.value) //filter options just in case the default value is included
-                .map((option, index) => (
-                <button
-                  type="button"
-                  css={optionStyle(option, selectedItem)(theme)}
-                  key={`${option.value}-${index}`}
-                  onClick={() => {handleSelect(option);}}
-                >
-                  {option.label}
-                </button>
-              ))}
+          <div css={menuStyle()(theme)} data-testid="filter-menu">
```

Comment:
Maybe you could add the `dataTestId` as a prefix here? Can this component be used multiple times in the same page?